backend/ml/management/commands/ml_train.py

```diff
@@ -79,6 +80,10 @@ def save_data(video_scores, contributor_rating_scores):
     Saves in the scores for Entities and ContributorRatings
     """
     EntityCriteriaScore.objects.all().delete()
+    for video_id, criteria, score, uncertainty in video_scores:
+       video=EntityCriteriaScore(entity_id=video_id).entity
+       video.tournesol_score+=10*score
```

Comment:
You should use some plugin within your IDE to highlight syntax issues. Install at least `isort` and `pylint` or `flake8`.
Details of unsuccessful checks cqn be seen in logs of Gihub CI.
Here you must insert spaces around arithmetic operators.

Reply:
If it helps, there is a section in the back end's `README.md` describing how to run locally the code quality checks.

Here:
https://github.com/tournesol-app/tournesol/blob/main/backend/README.md#code-quality

With docker the command is:
```shell
docker exec tournesol-dev-api scripts/ci/lint.sh
```

The output can be impressive at first, because there are a lot of warnings ( not errors ) we should take care of, or just disable. You can leave those warnings. What's relevant for you is to check if you see `chk1=1`, `chk2=1` (these are in the top 5 lines of the output), or `chk3=1` (this one is at the end), the digit 1 means one check has failed and the output will describe the problem.

Here is an example of error, viewed directly from GitHub, you need to click on the Python Code Quality Checks section, then navigate to the line 10:
https://github.com/tournesol-app/tournesol/runs/5289368490#step:6:10